### DISCUSSION TITLE
ImportError: attempted relative import with no known parent package

### DISCUSSION THREAD

**Discussion by MaximilianoVidal** [kevinzg/facebook-scraper#1092] [Q&A]
So I've been changing to absolute path, running it from console and everything but I keep getting the same message. 
No idea what to do

**Comment by ENGRZULQARNAIN** [comment]
This is my application structure using fastapi framework 
├── app
│   ├── __init__.py
│   ├── main.py
│   └── routers
│   │   ├── __init__.py
│   │   ├── blogs.py
│   │   └── users.py
│   └── internal
│       ├── __init__.py
│       └── admin.py

and I also get the same error

    from .routers import users
ImportError: attempted relative import with no known parent package        any one how to solve it
